option.go

```diff
@@ -241,26 +247,46 @@ func WithJwkErrRetryInterval(i string) Option {
 	access token parameters
 */
 
-// WithATEnableMTLSCertificateBoundAccessToken returns a EnableMTLSCertificateBoundAccessToken functional option
-func WithATEnableMTLSCertificateBoundAccessToken(b bool) Option {
+// WithNewATProcessor returns a functional option that appends the new access token processor parameters
+func WithNewATProcessorParam(verifyCertThumbprint bool, certBackdateDur, certOffsetDur string) Option {
```

Comment:
Why does the function name have "New" in it?

Reply:
it is append...

some alternatives:
1. `WithATProcessorParams(params ...ATProcessorParam)`, user need to care the ATProcessorParam struct
2. `NewATProcessorParam` + `WithATProcessorParams(params ...ATProcessorParam)`
3. `WithPushATProcessorParam` + `WithPopATProcessorParam`, push + pop

Reply:
ok understand.
I will implement 1 or 2.